.gitmodules

```diff
@@ -1,9 +1,11 @@
 [submodule "spack"]
   path = spack
-  #url = https://github.com/spack/spack
-  #branch = develop
+  ##url = https://github.com/spack/spack
+  ##branch = develop
+  #url = https://github.com/NOAA-EMC/spack
+  #branch = release/jcsda_emc_release_v1
```

Comment:
Please remove commented out code, unless you still need these for experimentation. Thanks!

Reply:
This will be reverted once the testing is complete, the spack PR is merged and the submodule pointer updates. (This is how git submodules work.)

Reply:
But we should probably remove the standing comment that we ultimately want to point to the authoritative spack repo at some point.